src/components/Router/index.js

```diff
@@ -291,6 +291,17 @@ class HistoryRouter extends Component {
     this.props.options.events.emit('complete', data)
   }
 
+  triggerOnError = (apiResponse) => {
+    const { status, response } = apiResponse
+    if (status === 0) return
+    const apiError = response.error || {}
+    const isExpiredTokenError = status === 401 && apiError.type === 'expired_token'
+    const type = isExpiredTokenError ? 'expired_token' : 'exception'
+    const message = apiError.message
+    this.props.options.events.emit('error', { type, message })
+    trackException(`${type} - ${message}`)
```

Comment:
Minor: This probably should be called `trackError` since there is now an error type called "exception" but it tracks both/any type of error.

Reply:
This is an method in the Tracker component that we've used to track all types of errors/exceptions